lib/matplotlib/font_manager.py

```diff
@@ -1278,12 +1278,15 @@ def _rebuild():
 try:
     fontManager = json_load(_fmcache)
 except Exception:
+    print("Rebuilding font cache.")
     _rebuild()
 else:
     if getattr(fontManager, '_version', object()) != FontManager.__version__:
         _rebuild()
     else:
+        print("Reusing font cache.")
         _log.debug("Using fontManager instance from %s", _fmcache)
+json.dump(fontManager, sys.stdout, cls=JSONEncoder, indent=2)
```

Comment:
Can you please add a comment how this is working? We're dumping the fontmanager to stdout at import time?!?

Reply:
Same as above.